src/components/fields/location-field/location-helper.ts

```diff
@@ -143,6 +143,28 @@ export namespace LocationHelper {
 		}
 	};
 
+	export const checkAndSetPinLocationAsResult = (query: string): IResultsMetaData => {
+		const [lat, lng] = query
+			.split(":")[1]
+			.split(",")
+			.map((value) => parseFloat(value.trim()));
```

Comment:
no need to trim before parseFloat

this function only work with a specific string format otherwise it will crash. it'll be a good idea to check the format via `hasGotPinLocationValue()` before running it